test/utils/exportUtils/PlainTextExport-test.ts

```diff
@@ -14,26 +14,59 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import { MatrixEvent, Room } from "matrix-js-sdk/src/matrix";
+
 import { createTestClient, mkStubRoom, REPEATABLE_DATE } from "../../test-utils";
 import { ExportType, IExportOptions } from "../../../src/utils/exportUtils/exportUtils";
 import PlainTextExporter from "../../../src/utils/exportUtils/PlainTextExport";
+import SettingsStore from "../../../src/settings/SettingsStore";
+
+class TestablePlainTextExporter extends PlainTextExporter {
+    public async testCreateOutput(events: MatrixEvent[]): Promise<string> {
+        return this.createOutput(events);
+    }
+}
 
 describe("PlainTextExport", () => {
+    let stubOptions: IExportOptions;
+    let stubRoom: Room;
     beforeEach(() => {
         jest.useFakeTimers();
         jest.setSystemTime(REPEATABLE_DATE);
-    });
-
-    it("should have a Matrix-branded destination file name", () => {
         const roomName = "My / Test / Room: Welcome";
         const client = createTestClient();
-        const stubOptions: IExportOptions = {
+        stubOptions = {
             attachmentsIncluded: false,
             maxSize: 50000000,
         };
-        const stubRoom = mkStubRoom("!myroom:example.org", roomName, client);
+        stubRoom = mkStubRoom("!myroom:example.org", roomName, client);
+    });
+
+    it("should have a Matrix-branded destination file name", () => {
         const exporter = new PlainTextExporter(stubRoom, ExportType.Timeline, stubOptions, () => {});
 
         expect(exporter.destinationFileName).toMatchSnapshot();
     });
+
+    it.each([
+        [24, false, /^\d{1,2}\/\d{1,2}\/\d{4},\s\d{1,2}:\d{2}:\d{2}\s-\s/],
```

Comment:
Since the event timestamp is fixed can you use a string here without regex comparison? It would highly improve the readability of the test.